### DISCUSSION TITLE
vendor/symfony/http-foundation/BinaryFileResponse.php on line 302 after update from livewire v2.10.6 -> 2.10.7

### DISCUSSION THREAD

**Discussion by benaf1** [livewire/livewire#5176] [7. Bugs (archived)] [answered]
Ran composer update.  It updated my livewire version from 2.10.6 -> 2.10.7.  Then I get a 500 error when trying to load /livewire/livewire.js?id=de3fca26689cb5a39af4.  And when I go to that url {site.com}/livewire/livewire.js?id=blahblah it is giving me this error -> Error: Call to undefined function Symfony\Component\HttpFoundation\ignore_user_abort() in file laravel/vendor/symfony/http-foundation/BinaryFileResponse.php on line 302

**Comment by benaf1** [comment]
*Note downgraded back to livewire v2.10.6 and works fine again.

**Comment by joshhanley** [comment]
@benaf1 ah interesting, 2.10.7 is working happily for me! Do you publish your javascript assets (I don't recommend doing that) ? If so did you re-publish them after the update? Also what version of Laravel are you using? Thanks!

**Reply by benaf1** [reply]
@joshhanley I'm on version 9.24.0 on laravel, php v8.0.21.  I don't think I published the javascript assets.

**Comment by benaf1** [comment, marked as answer]
Problem ended up being an update to symfony when ran the command composer update that seems to have required ignore_user_abort().  And I'm managing the server with runcloud.  And in the setting of the app there was a list of disabled php functions and of course ignore_user_abort was one of the disabled functions.  As soon as I enabled that func everything worked again.

**Reply by joshhanley** [reply]
@benaf1 ah yep, thanks for sharing!

**Reply by shadowsah97** [reply]
thank you so much
